ui/app/scripts/config.js

```diff
@@ -1,9 +1,6 @@
 
     var CONFIG = {
-        appUrl: document.location.protocol + "//" + document.location.host + "/api/",
-//        appUrl: 'http://localhost:8081/',
-        wsUrl: document.location.protocol == "https:" ? "wss:"  + "//" + document.location.host + "/ws/" : "ws:" + "//" + document.location.host + "/ws/",
-//        wsUrl: 'ws://localhost:8082/',
-//        hawtioUrl: 'http://localhost:8080/'
-        hawtioUrl: document.location.protocol + "//" + document.location.host + "/hawtio/"
+        appUrl: 'http://localhost:8081/',
+        wsUrl: 'ws://localhost:8082/',
+        hawtioUrl: 'http://localhost:8080/'
```

Comment:
you accidentally changed the config file

Reply:
I do not see this config file in my project.

Reply:
ui/app/scripts/config.js

Reply:
pushed